repos/system_upgrade/el7toel8/actors/removeupgradebootentry/libraries/library.py

```diff
@@ -33,6 +31,6 @@ def get_upgrade_kernel_filepath():
         api.current_logger().warning('Unexpectedly received more than one BootContent message.')
     if not boot_content:
         raise StopActorExecutionError('Could not create a GRUB boot entry for the upgrade initramfs.',
-                                      details={'details': 'Did not receive a message about the leapp-provided'
+                                      details={'details': 'Did not receive a message about the leapp-provided '
```

Comment:
Wouldn't be better to put the empty space into the beggining of the next chunk of the string? In case of more chunks, it is easier to catch missing space when it is in one column instead of various columns in the end. See:
```python
my string('somethin s.dfsdmf dk '
                'somethin s.dfsdmf dk lskjdfl skjdlkfj '
                'somethin '
                 ....)
```
vs
```python
my string('somethin s.dfsdmf dk'
                ' somethin s.dfsdmf dk lskjdfl skjdlkfj'
                ' somethin'
                 ....)
```